src/libs/ReactI13n.js

```diff
@@ -34,40 +32,24 @@ if ('client' === ENVIRONMENT) {
 var ReactI13n = function ReactI13n (options) {
     debug('init', options);
     options = options || {};
-    this._i13nNodeClass = 'function' === typeof options.i13nNodeClass ? options.i13nNodeClass : I13nNode;
```

Comment:
Yes it was originally for internal potential usage, and we don't document it, should be ok to be removed. will note this in the release note.